pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -17,6 +17,7 @@ func NewResourceSyncController(
 	eventRecorder events.Recorder) (*resourcesynccontroller.ResourceSyncController, error) {
 
 	resourceSyncController := resourcesynccontroller.NewResourceSyncController(
+		"KubeAPIServerOperator",
```

Comment:
not sure what is the preferred naming

it differs to what we already have here: https://github.com/openshift/cluster-kube-apiserver-operator/blob/48d9579a5c4d0f6eb9ddbbc96cb44ccda121ce09/pkg/operator/starter.go#L212

Reply:
seems that we go with the operand name: https://github.com/openshift/cluster-kube-apiserver-operator/pull/1747/commits/6f3ed3bc9b991818a5203c387bed69c32387fd0b